manifests/reposync-crd.yaml

```diff
@@ -421,6 +421,26 @@ spec:
                           x-kubernetes-int-or-string: true
                       type: object
                     type: array
+                  roleRefs:
```

Comment:
i see the `roleRefs` addition here, but shouldn't the `roleRef` have been removed? I can't find it...

Reply:
oh, nvm. this is reposync

Reply:
The `ClusterRole` api from the recent change was only added for RootSync.